ldk/node/src/hostClients/networkService.ts

```diff
@@ -8,7 +8,15 @@ export interface HttpRequest {
 
 export interface HttpResponse {
   statusCode: number;
-  data: string | Uint8Array;
+  /**
```

Comment:
Does this show up in the generated docs and code completion? If you don't know right off-hand I can pull it down and see.

Reply:
I think it does